src/main/java/frc/robot/Limelight.java

```diff
@@ -22,4 +62,202 @@ public void togglePipeline() {
         }
 
     }
+
+    /**
+     * Gets the horizontal offset angle from networkTable
+     * @return The horizontal offset angle from the limelight to the target
+     */
+    public Double getHorizontalOffset() {
+        
+        if(hasTargets()){
+
+        return table.getEntry("tx").getNumber(0).doubleValue();
+        
+        }
+        return Double.NaN;
+    }
+
+   
+
+    /**
+     * Gets the vertical offset angle from the limelight to the target
+     * @return The vertical angle from the limelight to the target
+     */
+    public Double getVerticalOffset() {
+
+        if(hasTargets()){
+
+            return table.getEntry("tx").getNumber(0).doubleValue();
+
+        }
+            return Double.NaN;
+
+    }
+
+   
+
+     /**
+      * Gets the skew of the robot from network tables
+      * @return The skew of the robot from -90 to 90 degrees
+      */
```

Comment:
i believe that the entry `ts0` represents the skew of the target in the viewframe rather than the robot, considering it is under the contour detection section in docs rather than robot pose